### DISCUSSION TITLE
Python 網頁爬蟲到底是什麼？

### DISCUSSION THREAD

**Discussion by vchien03** [1101-GS4719-C/Python-Community-GS4719-C#1592] [General]
相信大家多少都聽過「爬蟲」這個詞吧？
利用程式與資料爬蟲收集資料，是目前資料來源的一個重要的管道。
但是爬蟲是如何進行的呢？
以下這篇文章簡單地描述了爬蟲的流程，針對「網頁收集的開發工具生態系」、「學習資料爬蟲的幾個階段」以及「打造爬蟲是資料人的基本技能」三個段落，介紹 Python 網頁爬蟲的學習地圖與策略。
https://www.technice.com.tw/opinion/22406/
分享給大家~

**Comment by poyen0806** [comment]
這篇文章真的很實用，尤其對想入門資料爬蟲的人來說是個很好的指引！爬蟲的確是資料收集的重要技能，尤其在大數據時代，會寫爬蟲就像有了一把通往資料世界的鑰匙。

除了文章提到的工具和策略，我覺得可以再補充幾點學習方向：
1. 爬蟲的合法性與道德性
2. 爬蟲效能優化
3. 資料清理與存儲

這些可以問ncuedu的GPT
會回答得很清楚
